apps/core/priv/repo/migrations/20180213064751_create_user.exs

```diff
@@ -0,0 +1,46 @@
+defmodule Core.Repo.Migrations.CreateUser do
+  use Ecto.Migration
+
+  def change do
+    create table(:snitch_users) do
+      add :first_name, :string # Just a name perhaps?
+      add :last_name, :string
+      add :email, :string
+      add :password_hash, :string, comment: "Generated by argon2_elixir:hash_pwd_salt with default 16 byte salt length."
+      add :is_admin?, :boolean, default: false
+
+      add :sign_in_count, :integer, default: 0
+      add :failed_attempts, :integer, default: 0
+
+      # add :snitch_api_key, :string
+      # add :remember_token, :string
+
```

Comment:
Don't understand why these are even in place commented; either they go in the migration or don't. @oyeb